### PR TITLE
Roundels: optimise SVGs

### PR DESCRIPTION
**Pull request by mxdvl**
## What is the purpose of this change?

Optimise the SVG roundels’ code that was introduced in #579.

## What does this change?

- use brand colours
- reduce precision to 2 decimals
- use circle where relevant

### CODE REVIEW

File: src/core/brand/roundel-brand.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react"
 import { SvgRoundelFilled } from "./roundel-filled"
+import { brand } from "@guardian/src-foundations/palette"
```
Comment (SiAdcock) [inline src/core/brand/roundel-brand.tsx:3]: This is a great idea! Would it be possible for you to add `@guardian/src-foundations` as a `peerDependency` of this package?

Comment (mxdvl) [reply src/core/brand/roundel-brand.tsx:3]: Thanks, I’ll have a look at that!

File: src/core/brand/roundel-filled.tsx [hunk 2]
```diff
@@ -9,16 +9,10 @@ export const SvgRoundelFilled = ({
...
+			<circle cx="21" cy="21" r="21" fill={background} />
 			<path
-				fillRule="evenodd"
-				clipRule="evenodd"
-				d="M21.0001 0C9.4022 0 0 9.40185 0 21C0 32.598 9.4022 42 21.0001 42C32.5981 42 42 32.598 42 21C42 9.40185 32.5981 0 21.0001 0Z"
-				fill={background}
-			/>
-			<path
-				fillRule="evenodd"
-				clipRule="evenodd"
```
Comment (SiAdcock) [inline src/core/brand/roundel-filled.tsx:20]: Lazy Q: fillRule and clipRule get included by Figma when exporting SVGs. Are they doing anything useful, or can we just delete them every time?

**Review by SiAdcock — APPROVED** [review]
Thanks so much @mxdvl this looks amazing! Please could you squash and merge?

File: src/core/brand/roundel-filled.tsx (hunk 2 above)
Comment (mxdvl) [reply src/core/brand/roundel-filled.tsx:20]: They can be useful in complex cut-out shapes.

**Pull request merged**